src/operators/test_identity.py

```diff
@@ -5,31 +5,50 @@
 Identity operators are used to compare the objects, not if they are equal, but if they are actually
 the same object, with the same memory location.
 """
-
+import copy
 
 def test_identity_operators():
     """Identity operators"""
 
     # Let's illustrate identity operators based on the following lists.
     first_fruits_list = ["apple", "banana"]
     second_fruits_list = ["apple", "banana"]
-    third_fruits_list = first_fruits_list
+    third_fruits_list = first_fruits_list   #Shallow copy
 
     # is
     # Returns true if both variables are the same object.
 
     # Example:
     # first_fruits_list and third_fruits_list are the same objects.
     assert first_fruits_list is third_fruits_list
+    assert first_fruits_list == third_fruits_list
+
+
+    third_fruits_list[0] = "Mango"
+    print(first_fruits_list)
+
+    assert first_fruits_list is third_fruits_list
+    assert first_fruits_list == third_fruits_list
+
+
+    third_fruits_list = copy.deepcopy(first_fruits_list)   #Deep copy copy
+    # third_fruits_list[0] = "Mango"
```

Comment:
Could you remove this comment

Reply:
Yes, I can remove it. And now I have uncommented it.